### ISSUE TITLE
Linux release binary identifies as "canary" for v1.11.3

### ISSUE THREAD

**Issue by jsejcksn** [denoland/deno#11179]
Today when upgrading Deno to `1.11.3` on two devices, I noticed that the result of calling `deno --version` on Linux returns info about a `canary` release channel instead of `release`. See outputs below for comparison:

Linux:
```
$ deno --version
deno 1.11.3+f4bed10 (canary, x86_64-unknown-linux-gnu)
v8 9.1.269.35
typescript 4.3.2
```

macOS:
```
% deno --version
deno 1.11.3 (release, x86_64-apple-darwin)
v8 9.1.269.35
typescript 4.3.2
```

This discrepancy seems like a bug, but if it's intentional and documented somewhere, please feel free to point me to the documentation and close.

**Comment by piscisaureus** [comment]
This is a bug in the github actions workflow file.
(https://github.com/denoland/deno/blob/5db9f627e3b105ffc1ed43bb81732886308ddfdf/.github/workflows/ci.yml#L215)

**Comment by lucacasonato** [comment]
Still an issue on 1.11.4

**Comment by jsejcksn** [comment]
Seems to be fixed in `1.11.5`
